README.MD

```diff
@@ -284,7 +285,7 @@
 ## <a name="W"> </a>W
 
 - [Waihona](https://github.com/bisohns/waihona) - Rust crates for performing cloud storage actions across AWS, GCP and Azure. **By [@deven96](https://twitter.com/_deven96)**
-- [WatchDog](https://github.com/IzyPro/WatchDog) - WatchDog is a Realtime HTTP (Request & Response) and Exception logger and viewer for ASP.Net Core Web Apps and APIs. **By [@IzyPro_](https://twitter.com/IzyPro_)**
+- [WatchDog](https://github.com/IzyPro/WatchDog) - WatchDog is a Realtime HTTP (Request & Response) and Exception logger and viewer for ASP.Net Core Web Apps and APIs. **By [@IzyPro\_](https://twitter.com/IzyPro_)**
```

Comment:
Is this a typo? the slash in the author's name.

Reply:
I have no idea. Probably a mistake from me. I can't really remember.

Reply:
Would have to update it so I can merge.

Reply:
Yep. I'm on it.